src/pages/content-edit/form.css

```diff
@@ -106,25 +90,55 @@
     font-weight: normal;
 }
 
-.title {
+input[type=text].title {
     margin: 0;
     padding: 2px 5px 0 5px;
-    
-    border: solid 1px transparent;
-    border-bottom: solid 2px #68A;
+
+    border: none;
+    border-bottom: solid 3px #68A;
+    border-radius: 0;
+    box-shadow: none;
+    font-size: 2em;
+    padding: 0;
+    width: 100%;
 }
 
-.title:hover {
+input[type=text].title:hover {
```

Comment:
Either is a fine, add a comment explaining why though. Otherwise it leaves it to the imagination.